.github/workflows/create-release.yml

```
@@ -0,0 +1,53 @@
+name: Create release candidate
+
+on:
+  workflow_dispatch:
+    inputs:
+      releaseLevel:
+        description: 'Release level: major, minor, or patch.'
+        required: true
+        default: 'patch'
+
+jobs:
+  create_release:
+    name: Create release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          fetch-depth: 0
+
+      - name: Run preparation script
+        run: bash .github/scripts/prepare_release.sh ${{ github.event.inputs.releaseLevel }}
```

FYI there's no need to explicitly specify this runs via bash (you have a shebang in the file).

```suggestion
        run .github/scripts/prepare_release.sh ${{ github.event.inputs.releaseLevel }}
```